tools/cover-by-test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -x
+
+for testfile in $(find t -name "*.t"); do
```

Comment:
The easiest way around in this case is probably:
```suggestion
shopt -s globstar nullglob
for testfile in t/**/*.t; do
```

Reply:
Maybe something like this could also work in `sh`:

```
find t -name "*.t" -print0 | while IFS= read -r -d '' testfile
do
    echo "===== $testfile"
    rm -rf cover_db
    ./tools/invoke-tests --coverage "$testfile"
    cover -report codecovbash
    perl tools/cover-by-test.pl "$testfile" cover_db/codecov.json cover-by-test
done
```